apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+$red: #ff0000;
+
+.nav {
+	background-image: url("../../assets/images/lined_paper.svg");
+	background-size: cover;
+	background-position: left top;
+	background-repeat: no-repeat;
+}
+
+@media screen and (max-width: 576px) {
+	.nav {
+		background-size: contain;
+		background-repeat: repeat-y;
+	}
+}
+
+.home,
+.resources,
+.schedule {
+	margin-left: 10px;
+	margin-right: 10px;
+	font-size: 36px;
+	font-weight: bold;
+	color: #000000;
+	line-height: normal;
+}
+
+.home:hover {
+	color: variables.$green;
+}
+
+.resources:hover {
+	color: variables.$blue;
+}
+
+.schedule:hover {
+	color: $red;
+}
```

Comment:
Apologies for not clarifying this for you earlier, but instead of the color changing on hover, the link for the active page should be the corresponding color. For example, if we're on the home page, then the color for the "Home" link should be green instead of the default.

Reply:
should i leave the hover in?

Reply:
No, it's no longer necessary now that the actual color and hover colors are the same.